.github/workflows/container-image-build.yml

```diff
@@ -27,6 +27,16 @@ on:
         description: "Whether to push the image afterwards"
         type: boolean
         default: true
+      multiPlatform:
+        required: false
+        description: "Enable multiPlatform builds for the container image"
+        type: boolean
+        default: false
+      platform:
+        required: false
+        description: "Architecture platforms for the image build (linux/amd64,linux/arm64, etc.)"
+        type: string
+        default: ""
```

Comment:
I am thinking if we would need a default value here

Reply:
I _guess_ (honestly, I didn't tested it) that if you don't set `multiPlatform: false` this will be ignored and built against the arch where it has been executed (`linux/amd64` by default IIRC)

Reply:
I _guess_ (honestly, I didn't tested it) that if you don't explicitly set `multiPlatform: true` this will be ignored and built against the arch where it has been executed (`linux/amd64` by default IIRC)